source/cl/scripts/cts-3.0-online-ignore-linux-riscv.csv

```diff
@@ -1,1310 +1,3 @@
-Pipes,pipes/test_pipes pipe_convenience_readwrite_uint
-Pipes,pipes/test_pipes pipe_convenience_readwrite_long
-Pipes,pipes/test_pipes pipe_convenience_readwrite_ulong
-Pipes,pipes/test_pipes pipe_convenience_readwrite_short
-Pipes,pipes/test_pipes pipe_convenience_readwrite_ushort
-Pipes,pipes/test_pipes pipe_convenience_readwrite_float
-Pipes,pipes/test_pipes pipe_convenience_readwrite_half
-Pipes,pipes/test_pipes pipe_convenience_readwrite_char
-Pipes,pipes/test_pipes pipe_convenience_readwrite_uchar
-Pipes,pipes/test_pipes pipe_convenience_readwrite_double
-Pipes,pipes/test_pipes pipe_convenience_readwrite_struct
-Pipes,pipes/test_pipes pipe_info
-Pipes,pipes/test_pipes pipe_max_args
-Pipes,pipes/test_pipes pipe_max_packet_size
-Pipes,pipes/test_pipes pipe_max_active_reservations
-Pipes,pipes/test_pipes pipe_query_functions
-Pipes,pipes/test_pipes pipe_readwrite_errors
-
-SVM,SVM/test_svm svm_byte_granularity
-SVM,SVM/test_svm svm_set_kernel_exec_info_svm_ptrs
-SVM,SVM/test_svm svm_fine_grain_memory_consistency
-SVM,SVM/test_svm svm_fine_grain_sync_buffers
-SVM,SVM/test_svm svm_shared_address_space_fine_grain
-SVM,SVM/test_svm svm_shared_sub_buffers
-SVM,SVM/test_svm svm_shared_address_space_fine_grain_buffers
-SVM,SVM/test_svm svm_allocate_shared_buffer
-SVM,SVM/test_svm svm_shared_address_space_coarse_grain_old_api
-SVM,SVM/test_svm svm_shared_address_space_coarse_grain_new_api
-SVM,SVM/test_svm svm_cross_buffer_pointers_coarse_grain
-SVM,SVM/test_svm svm_pointer_passing
-SVM,SVM/test_svm svm_enqueue_api
-SVM,SVM/test_svm svm_migrate
-
-Device timer,device_timer/test_device_timer timer_resolution_queries
-Device timer,device_timer/test_device_timer device_and_host_timers
-SPIRV new,spirv_new/test_spirv_new ext_cl_khr_spirv_no_integer_wrap_decoration_fadd_int
-SPIRV new,spirv_new/test_spirv_new ext_cl_khr_spirv_no_integer_wrap_decoration_fsub_int
-SPIRV new,spirv_new/test_spirv_new ext_cl_khr_spirv_no_integer_wrap_decoration_fmul_int
-SPIRV new,spirv_new/test_spirv_new ext_cl_khr_spirv_no_integer_wrap_decoration_fshiftleft_int
-SPIRV new,spirv_new/test_spirv_new ext_cl_khr_spirv_no_integer_wrap_decoration_fnegate_int
-SPIRV new,spirv_new/test_spirv_new ext_cl_khr_spirv_no_integer_wrap_decoration_fadd_uint
-SPIRV new,spirv_new/test_spirv_new ext_cl_khr_spirv_no_integer_wrap_decoration_fsub_uint
-SPIRV new,spirv_new/test_spirv_new ext_cl_khr_spirv_no_integer_wrap_decoration_fmul_uint
-SPIRV new,spirv_new/test_spirv_new ext_cl_khr_spirv_no_integer_wrap_decoration_fshiftleft_uint
-SPIRV new,spirv_new/test_spirv_new op_vector_times_scalar_double
-
-SPIR (cl_khr_spir),spir/test_spir conversions_double no-unzip
-Mipmaps (Kernel),images/kernel_read_write/test_image_streams test_mipmaps CL_FILTER_NEAREST
-Mipmaps (clCopyImage),images/clCopyImage/test_cl_copy_images
-Mipmaps (clReadWriteImage),images/clReadWriteImage/test_cl_read_write_images test_mipmaps
+Select,select/test_select
+Math,math_brute_force/test_bruteforce -w
```

Comment:
do we really not want to run bruteforce at all now? Is it not feasible to split this up now? same goes for test_select.

Reply:
If test_select and test_bruteforce take options that let us exclude the subtests that take too long, we could use that, and make our cityrunner to accept multiple CSV files so that we can say "use upstream quick.csv, except ignore-riscv.csv, but add custom-riscv.csv". I think that would need a cityrunner update in a separate PR, I don't think we should include it in here, but if we decide to do that, we could merge that first and avoid any window where we ignore these tests.

Reply:
fair enough, I was just worried that we miss important accuracy fails. Maybe a jira to look into whether we should restore some of what we've lost

Reply:
Actually, I had a thought about this now, I think we can cover these with minimal effort. Will update.

Reply:
Have updated, hope to verify today that the changes to city_runner work. Can pull those to a separate PR if we prefer.